app/data_model/database.py

```diff
@@ -60,6 +60,19 @@ def __repr__(self):
         return "<EQSession('%s', '%s', '%s')>" % (self.eq_session_id, self.user_id, self.timestamp)
 
 
+class UsedToken(base):
+    __tablename__ = "used_token"
+    token = Column("token", String, primary_key=True)
```

Comment:
except we're adhering to our ONS schema spec, so it should be a jti claim not just any value.